packages/components/navlist/src/NavList/NavList.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { cx } from 'emotion';
+import {
+  type CommonProps,
+  type PolymorphicProps,
+  type PolymorphicComponent,
+  type ExpandProps,
+} from '@contentful/f36-core';
+import { getStyles } from './NavList.styles';
+
+const NAV_LIST_DEFAULT_TAG = 'div';
+
+export interface NavListInternalProps extends CommonProps {
+  children: React.ReactNode;
+  /**
+   * @default div
+   */
+  as?: 'nav' | 'div';
+}
+
+export type NavListProps<
+  E extends React.ElementType = typeof NAV_LIST_DEFAULT_TAG,
+> = PolymorphicProps<NavListInternalProps, E>;
+
+function _NavList<E extends React.ElementType = typeof NAV_LIST_DEFAULT_TAG>(
+  props: NavListProps<E>,
+  ref: React.Ref<any>,
+) {
+  const styles = getStyles();
+
+  const { as, className, children, testId, ...otherProps } = props;
+  const role = as === 'nav' ? 'navigation' : undefined;
+
+  const Element = (as ?? NAV_LIST_DEFAULT_TAG) as React.ElementType;
+
+  return (
+    <Element
+      aria-label="Sidebar"
```

Comment:
```suggestion
      aria-label="Sidebar Navigation"
```
We probably should expose it as a prop as well. users should be able to provide descriptive text about the menu

Reply:
Should we add a custom prop tho?? because the user can already pass an `aria-label` props, that's why I even added it before the `{...otherProps}`.
And adding another prop we would have the custom prop and also the `aria-label`, which should be the priority in this case? 🤔

Reply:
Ah yes you're right it'll be overridden, I missed that, thanks 👍🏼